sources/tools/Stride.VisualStudio.Package/Stride.VisualStudio.Package.nuspec

```diff
@@ -9,8 +9,8 @@
     <projectUrl>http://stride3d.net/</projectUrl>
     <iconUrl>https://raw.githubusercontent.com/stride3d/media/master/images/mainlogo/nuget/logo.png</iconUrl>
     <repository type="git" url="https://github.com/stride3d/stride" />
-    <copyright>&#169; Stride contributors and Silicon Studio Corp.</copyright>
-    <description>Visual Studio Integration Plugin for Stride</description>
+    <copyright>&#169; Stride contributors and  the .NET Foundation</copyright>
+    <description>Visual Studio Extension for Stride</description>
```

Comment:
Same as copyright issue mentioned previously, probably should including contributors, dotnet and Silicon Studio